layouts/partials/inline/inline.html

```diff
@@ -38,4 +40,30 @@
       '<div class="tvp-videos-scroller" id="tvpVideoScroller">' +
       '</div>' +
    '</div>' +
-'</div>'
\ No newline at end of file
+'</div>',
+'loader': '<div id="tvp_widget_loader">'+
```

Comment:
The idea was to have both "skeleton" and "skeleton CSS" inlined in the embed code, they are not currently added there

![selection_172](https://user-images.githubusercontent.com/8630605/31971380-706c2126-b8d0-11e7-8381-3e7c43a4090d.png)

**This should be something like:**

```
<style>CSS needed for the skeleton</style>
<div id="inline-2">HTML needed for the skeleton</div>
<script>
```

Therefore, the [widget's embed code template file](https://github.com/tvpage/tvpage-hugo-template/blob/master/embed-templates/inline.tmpl) needs to be updated to include that css & html